src/main/java/com/example/demo/assembler/UserAssembler.java

```diff
@@ -0,0 +1,66 @@
+package com.example.demo.assembler;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.stereotype.Component;
+import org.springframework.util.CollectionUtils;
+
+import com.example.demo.model.User;
+import com.example.demo.response.dto.UserDto;
+import com.example.demo.response.dto.UserResponseDto;
+
+@Component
+public class UserAssembler {
+	
+	public UserResponseDto toModel(List<User> users, Optional<User> opUser)
+	{
+		UserResponseDto userResponse = new UserResponseDto();
+		UserDto userDto = new UserDto();
+		List<UserDto> userDtolist = new ArrayList<>();
+		
+		if(CollectionUtils.isEmpty(users) && opUser.isPresent()== true)
+		{
```

Comment:
- Don't use isEmpty logic here, their is no need to check here 

-  Use -> User user = opUser.get() and then use it

Reply:
``
	UserDto userDto = new UserDto();
	
	public UserDto toModel( Optional<User> opUser)
	{	
		User user = opUser.get();
		
		userDto.setName(user.getName());
		userDto.setDesignation(user.getDesignation());
		userDto.setDu(user.getDu());
		userDto.setExtension(user.getExtension());